src/config/dynamic-theme-fixes.config

```diff
@@ -4513,6 +4513,15 @@ CSS
 
 ================================
 
+docs.soliditylang.org
+
+CSS
+html {
+    background-color: unset !important;
```

Comment:
We shouldn't be touching the background-color of `html` in general. It's a important property for the browsers and used for initial rendering. I don't think that this is fixable.

Reply:
I can appreciate that rule of thumb. Pushing an alternate idea.